concrete/docs/getting_started/installation.md

```diff
@@ -52,10 +52,35 @@ concrete = { version = "0.2.0", features = [ "booleans", "shortints", "integers"
 
 ## Supported platforms
 
-As `concrete` relies on `concrete-core`, `concrete` is only supported on `x86_64 Linux` and `x86_64 macOS`.
+As `concrete` relies on `concrete-core`, `concrete` is supported on:
+  - `x86_64 Linux`
+  - `aarch64 Linux`
+  - `x86_64 macOS`
+  - `aarch64 macOS`
 
 Windows users can use `concrete` through the `WSL`.
```

Comment:
Windows is supported now theoretically

Reply:
yeah windows with rdseed should work, but we don't have the "windows seeder" fallback

Reply:
And its not in the CI (aarch64 macOs is not either 😳 )

Reply:
core is tested with the M1 mac, we haven't had time to propagate the CI infrastructure, but it's not too hard to do if it's required at some point